lib/expand.js

```diff
@@ -1148,7 +1150,13 @@ class Expander {
       // We didn't find it, so look in the unexpanded elements, then expand it
       const unexpanded = this._unexpanded.dataElements.findByIdentifier(identifier);
       if (typeof unexpanded !== 'undefined') {
-        return this.expandElement(unexpanded);
+        for (const file of this._unexpanded.dataElements.files) {
+          for (const de of this._unexpanded.dataElements.byFile(file)) {
+            if (de.identifier.equals(identifier)) {
```

Comment:
Ooof.  Is this the only way to find an element and its file by the element identifier?  It's fine -- but some day we may want to revisit this (and all the other places where we do things inefficiently).